### PR TITLE
added animations to the about us page

### PR DESCRIPTION
**Pull request by kartikeya-123**
I have made some changes as suggested by @smaranjitghose . Please check it.

### CODE REVIEW

**Review by anushbhatia — CHANGES_REQUESTED** [review]
text not visible in subheading please remove that

**Comment by kartikeya-123** [comment]
> text not visible in subheading please remove that
   I have made the suggested changes.

**Comment by anushbhatia** [comment]
Resolve conflict @abhu-A-J @ashishnagpal2498 merge them

**Comment by ashishnagpal2498** [comment]
Okay. I will merge it once @abhu-A-J will resolve the conflicts.

**Comment by anushbhatia** [comment]
@kartikeya-123 you can resolve the conflict

File: src/components/About.jsx [hunk 1]
```diff
@@ -1,117 +1,219 @@
...
-  
+  background-image: url(https://www.xmple.com/wallpaper/linear-white-gradient-highlight-blue-1920x1080-c2-ffffff-e0ffff-l-50-a-300-f-21.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+
+  display: flex;
+  width: 100%;
+  margin: 0px;
+  // box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.5);
```
Comment (ashishnagpal2498) [inline src/components/About.jsx:43]: Remove this comment

File: src/components/About.jsx [hunk 2]
```diff
@@ -1,117 +1,219 @@
...
 const Logos = styled.div`
-    display:flex;
-    
-    justify-content: stretch;
-    align-self: center;
-    border-top: 1px solid rgba(0,0,0,0.3);
-    
-   
-    padding: 2em;
-    /* height: 50em; */
```
Comment (ashishnagpal2498) [inline src/components/About.jsx:79]: This.

Comment (ashishnagpal2498) [reply src/components/About.jsx:79]: Remove all code comments from your code.

File: src/components/About.jsx [hunk 3]
```diff
@@ -1,117 +1,219 @@
...
+  display: flex;
```
Comment (ashishnagpal2498) [inline src/components/About.jsx:73]: No Need of these properties,
width , display , margin. You are mixing properties unnecessarily.

**Review by ashishnagpal2498 — CHANGES_REQUESTED** [review]

**Comment by ashishnagpal2498** [comment]
You are mixing a whole bunch of CSS properties, which are not required. Kindly open inspect and look for those properties and remove them. Self review your code first.

**Comment by ashishnagpal2498** [comment]
@kartikeya-123 . If you don't reply to the conversation. Unfortunately I have to close this PR.

**Pull request closed without merging**